### DISCUSSION TITLE
What do the dBm values in the console refer to?

### DISCUSSION THREAD

**Discussion by peff74** [hoylabs/OpenDTU-OnBattery#258] [Q&A] [answered]
I have recently had problems with disconnections from the inverter.
Too much foliage on the bushes^^
After shielding the nrf it worked again.

However, I have now noticed dBm value in the console, which I had not seen before?
![2023-06-13 08_40_54-OpenDTU](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/91614910/36f52475-f02f-4e01-9bb5-20c4b743d0e4)

But no matter what I do (increase power or go closer to the inverter) they always stay the same

**Comment by helgeerbe** [comment, marked as answer]
Well, I guess you should place your question in the original [OpenDTU repo](https://github.com/tbnobody/OpenDTU).

I checked my console and see the same -80dBM.

**Reply by peff74** [reply]
done --> [Link](https://github.com/tbnobody/OpenDTU/discussions/1031)
